back/src/main/java/com/cocktailpick/back/cocktail/dto/CocktailDetailResponse.java

```diff
@@ -39,7 +40,9 @@
 
 	private List<RecipeItemResponse> recipe;
 
-	public static CocktailDetailResponse of(Cocktail cocktail) {
+	private boolean favorite;
+
+	public static CocktailDetailResponse of(Cocktail cocktail, boolean isFavorite) {
```

Comment:
```suggestion
	public static CocktailDetailResponse of(Cocktail cocktail, boolean favorite) {
```

매개변수 이름 바꾸어도 될 듯 합니다.